### PR TITLE
update to latest teku libraries to support ropsten

### PR DESCRIPTION
**Pull request by jframe**
## PR Description
Add support for the ropsten beacon chain network

## Documentation

- [x] I thought about documentation and added the `doc-change-required` label to this PR if updates are required.

## Changelog

- [x] I thought about adding a changelog entry, and added one if I deemed necessary.

### CODE REVIEW

File: commandline/src/main/java/tech/pegasys/web3signer/commandline/subcommands/Eth2SubCommand.java [hunk 1]
```diff
@@ -51,7 +51,7 @@ public class Eth2SubCommand extends ModeSubCommand {
       names = {"--network"},
       paramLabel = "<NETWORK>",
       description =
-          "Predefined network configuration to use. Possible values: [mainnet, pyrmont, prater, kintsugi, kiln, gnosis, minimal], file path"
+          "Predefined network configuration to use. Possible values: [mainnet, pyrmont, prater, kintsugi, kiln, gnosis, minimal, ropsten], file path"
```
Comment (usmansaleem) [inline commandline/src/main/java/tech/pegasys/web3signer/commandline/subcommands/Eth2SubCommand.java:54]: I wonder if we should create a String constant from Teku's Eth2Network enum and use that in description. That would free us from modifying it every time we support a new network.

**Review by usmansaleem — APPROVED** [review]
LGTM

File: commandline/src/main/java/tech/pegasys/web3signer/commandline/subcommands/Eth2SubCommand.java (hunk 1 above)
Comment (siladu) [inline commandline/src/main/java/tech/pegasys/web3signer/commandline/subcommands/Eth2SubCommand.java:54]: Could also remove kintsugi to keep in line with teku: https://github.com/ConsenSys/teku/commit/b329dc05bce59c3cc484f260013ac584d71ebe40

Comment (jframe) [reply commandline/src/main/java/tech/pegasys/web3signer/commandline/subcommands/Eth2SubCommand.java:54]: Another reason to use the enum somehow

Comment (jframe) [reply commandline/src/main/java/tech/pegasys/web3signer/commandline/subcommands/Eth2SubCommand.java:54]: Good idea. I'll try and work how to use that with pico

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 ## Next version
 ### Features Added
 - Eth2 keystore bulk loading allowing a directory of keystores to be loaded without config files
+- Updated Teku libraries to provide support for `ropsten` network
```
Comment (siladu) [inline CHANGELOG.md:6]: Worth mentioning kintsugi support is removed here too?

**Review by siladu — APPROVED** [review]
LGTM

**Pull request merged**